ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -211,22 +211,19 @@ StatusModal {
             spacing: Style.current.padding
             Layout.fillWidth: true
             Layout.fillHeight: true
-            model: popup.channel? popup.channel.members : []
+            model: popup.store.currentChatContentModule().usersModule.model//popup.channel? popup.channel.members : []
```

Comment:
@borismelnik this list refers to the list of members of this group chat... 
- remove commented part `//popup.channel? popup.channel.members : []`
- updated this line to this: `model: popup.chatContentModule.usersModule.model`